src/map.h

```diff
@@ -40,8 +40,11 @@ class computer;
 struct itype;
 struct mapgendata;
 struct trap;
+struct oter_t;
+
 using trap_id = int_id<trap>;
-struct oter_id;
+using oter_id = int_id<oter_t>;
```

Comment:
> These aren't nice names.

It's not an ideal name, but it's perfectly consistent with other ids. What would you suggest instead?

> How badly would line count be affected by more descriptive naming?

Massively. I've counted 623 occurrences. I definitely don't want to to it here.